scripts/pre_commit_linter.py

```diff
@@ -1085,6 +1086,109 @@ def _check_directive_scope(all_files):
     return summary_messages
 
 
+class CustomHTMLParser(HTMLParser.HTMLParser):
+    """Custom HTML parser to check indentation."""
+
+    def __init__(self, filename, failed=False):
+        HTMLParser.HTMLParser.__init__(self)
+        self.failed = failed
+        self.filename = filename
+        self.previous_tag_line_number = None
+
+    def handle_starttag(self, tag, attrs):
+        line_number, column_number = self.getpos()
+        # Check the indentation of the tag.
+        if self.previous_tag_line_number is None:
+            if column_number % 2 != 0:
+                print (
+                    '%s --> Please ensure that %s tag '
+                    'in this file on line number %s '
```

Comment:
Rephrase the message to:
`Indentation level for <tag> tag on line <line number> should be a multiple of two.`

Reply:
Where should the filename be mentioned in the error message?

Reply:
Good catch, yes. Keep the filename and line arrow.